### PR TITLE
Feature/gh actions

### PR DESCRIPTION
**Pull request by turboladen**
Updates to get Actions running

### CODE REVIEW

**Review by agrian-joel — APPROVED** [review]

File: .github/workflows/rust-ci.yml [hunk 1]
```diff
@@ -27,8 +31,8 @@ jobs:
             ~/.cargo/git
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```
Comment (tindron) [inline .github/workflows/rust-ci.yml:33]: There is no Cargo.lock to hash

**Review by tindron — APPROVED** [review]

**Comment by agrian-joel** [comment]
Will you be adding a replacement for the release artifact step?

https://teamcity-build.agriandev.com/admin/editRunType.html?id=buildType:Crates_WiseUnits_Test&runnerId=RUNNER_176&cameFromUrl=%2Fadmin%2FeditBuildRunners.html%3Fid%3DbuildType%253ACrates_WiseUnits_Test%26init%3D1&cameFromTitle=

File: .github/workflows/rust-ci.yml (hunk 1 above)
Comment (turboladen) [reply .github/workflows/rust-ci.yml]: Switched to use Cargo.tomls, per https://stackoverflow.com/questions/64634535/github-actions-cache-rust-artifacts

**Comment by turboladen** [comment]
> Will you be adding a replacement for the release artifact step?
> 
> https://teamcity-build.agriandev.com/admin/editRunType.html?id=buildType:Crates_WiseUnits_Test&runnerId=RUNNER_176&cameFromUrl=%2Fadmin%2FeditBuildRunners.html%3Fid%3DbuildType%253ACrates_WiseUnits_Test%26init%3D1&cameFromTitle=

Still trying to figure out how to do this; will do as a separate PR.

**Pull request merged**